### PR TITLE
fix(api-reference): search modal a11y improvements

### PR DESCRIPTION
**Pull request by hwkr**
Adds screen reader support for the search modal including accessible names and aria descriptions.

### CODE REVIEW

File: packages/api-client/package.json [hunk 1]
```diff
@@ -77,11 +77,6 @@
       "types": "./dist/libs/importers/index.d.ts",
       "default": "./dist/libs/importers/index.js"
     },
-    "./libs/event-busses": {
-      "import": "./dist/libs/event-busses/index.js",
-      "types": "./dist/libs/event-busses/index.d.ts",
-      "default": "./dist/libs/event-busses/index.js"
-    },
```
Comment (antlio) [inline packages/api-client/package.json:84]: should we keep this in another commit/pr?

**Review by antlio — APPROVED** [review]
overall it looks great! 

could be done in another PR but we could also keep the accent color while tabbing, but maintain a darker shade of grey for focus. what do you think? cc @cameronrohani

**Comment by hwkr** [comment]
> overall it looks great! 
> 
> 
> 
> could be done in another PR but we could also keep the accent color while tabbing, but maintain a darker shade of grey for focus. what do you think? cc @cameronrohani 

That should be how it works! The search results are a little weird though because the don't actually gain focus via tab the way a normal element would, they us an aria active descendant pattern

**Review by marclave — APPROVED** [review]
awesome work @hwkr ! waiting on @cameronrohani to review

**Review by cameronrohani — APPROVED** [review]

**Comment by marclave** [comment]
@hwkr feel free to merge once conflicts are fixed : )

**Review by hanspagel — APPROVED** [review]
